content/blog/supervisory-authorities.fr.md

```diff
@@ -0,0 +1,62 @@
+{
+    "title": "Autorités de contrôle de la protection des données",
+    "type": "autorites-de-controle",
+    "url": "autorites-de-controle",
+    "aliases": [
+    	"supervisory-data-protection-authorities",
+    	"dpas",
+    	"svas"
+    ],
+    "date": "2019-02-07T16:58:19+02:00",
+    "description": "Les autorités de contrôle de la protection des données sont des structures indépendantes dont la tâche est de veiller au respect des lois sur la protection des données. Si tu penses qu'une entreprise enfreint tes droits, tu peux les contacter gratuitement. Nous t'expliquons ici quelles sont les autorités de contrôle et t'aidons à en trouver une à contacter si tu veux déposer une plainte.",
+    "featured_image": "/card-icons/authority.svg",
+    "tags": [ "plainte", "compétence" ],
+    "authors": [ "baltpeter" ],
```

Comment:
If you want, feel free to add yourself as an author so you're credited on the page. In that case, you would need to add an entry for you [here](https://github.com/datenanfragen/website/blob/master/config/_default/params.toml).

Reply:
Thanks, but I prefer to be credited as a translator somewhere on the website than as the author of this page :smile: I didn't write the content!